lib/src/components/NxToast/NxToast.scss

```diff
@@ -22,13 +23,15 @@
   right: 0;
   opacity: 1;
   overflow: hidden;
+  margin-top: var(--nx-spacing-2x);
 
   &.nx-toast--closing {
     animation-name: animationOut;
   }
 
   .nx-alert {
-    margin: var(--nx-spacing-2x) 0 0 0;
+    margin: 0;
+    pointer-events:auto;
```

Comment:
Let's put this on `nx-toast` to minimize the number of overrides that we're doing on an element that belongs to another component.

Reply:
> Let's put this on `nx-toast` to minimize the number of overrides that we're doing on an element that belongs to another component.

Moved into nx-toast here: [3deee00](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/3deee004bfc1d2460397ade2edc819b2cbbc6412)